Src/Microsoft.Dynamic/Microsoft.Dynamic.csproj

```diff
@@ -24,8 +24,4 @@
     <Reference Include="System.Runtime.Remoting" Condition=" $(Features.Contains('FEATURE_REMOTING')) " />
     <Reference Include="System.Xaml" Condition=" $(Features.Contains('FEATURE_WPF')) " />
   </ItemGroup>
-  <Import Project="$(BeforeTargetFiles)" />
-  <Target Name="BeforeBuild" DependsOnTargets="$(BeforeTargets);GenerateCurrentVersion" />
```

Comment:
We're still going to need these. We just have to change them to something like `<Target Name="BeforeBuildStart" BeforeTargets="BeforeBuild"`.